docs/developers/index.md

```diff
@@ -1,14 +1,25 @@
 # Table of Contents
 
-This section of the OpenTrials documentation is for developers. Here you can learn about the design of the platform, and how to get OpenTrials running locally or on your own servers, and the process for contributing enhancements and bug fixes to the code.
+This section of the OpenTrials documentation is for developers.
+Here you can learn about the design of the platform, how to use it and the process for contributing enhancements and bug fixes to the code.
 
-- [Getting started](#getting-started)
-- [Design](design/)
+- [Overview](#overview)
+- [Database model](data-model.md)
+- [API](api.md)
 
-# Getting started
+# Overview
 
-Get started.
+OpenTrials project has 4 main components:
+  - [Collectors](https://github.com/opentrials/collectors): contains logic for
+gathering data (ex: scrapers) and manages the schema for our `warehouse` database
+that keeps the data collected from different sources
+  - [Processors](https://github.com/opentrials/processors): contains logic for
+normalizing and enriching data in our `warehouse` and API `database` and manages our file storage
```

Comment:
Not sure how to deal with our `database` name here. We shouldn't expose it as a keyword, since it's not referenced as such anywhere in the code (if `warehouse` is defined by a `WAREHOUSE_URL`, the API DB is defined by a generic `DATABASE_URL` that doesn't sound at all like a project specific concept).

Following this rationale, I would remove all backticks from around "database" as it is not an OT specific keyword.